mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulHomeTest.kt

```diff
@@ -48,4 +48,29 @@ class StatefulHomeTest {
     rule.onNodeWithText(strings.sectionSocial).assertIsSelected()
     rule.onNodeWithText(strings.sectionSettings).assertIsNotSelected()
   }
+
+  @Test
+  fun clickPlaySection_selectsPlaySection() = runTest {
+    val api = AuthenticationFacade(emptyAuth(), emptyStore())
+    api.signUpWithEmail("email", "name", "password")
+    val user = api.currentUser.filterIsInstance<AuthenticatedUser>().first()
+    val strings = rule.setContentWithLocalizedStrings { StatefulHome(user) }
+    rule.onNodeWithText(strings.sectionPlay).performClick()
+    rule.onNodeWithText(strings.sectionPlay).assertIsSelected()
+    rule.onNodeWithText(strings.sectionSocial).assertIsNotSelected()
+  }
+
+  @Test
+  fun clickNewGame_inPlaySection_switchesToBoard() = runTest {
+    val api = AuthenticationFacade(emptyAuth(), emptyStore())
+    api.signUpWithEmail("email", "name", "password")
+    val user = api.currentUser.filterIsInstance<AuthenticatedUser>().first()
+    val strings = rule.setContentWithLocalizedStrings { StatefulHome(user) }
+    rule.onNodeWithText(strings.sectionPlay).performClick()
+    rule.onNodeWithText(strings.sectionPlay).assertIsSelected()
+    rule.onNodeWithText(strings.sectionSocial).assertIsNotSelected()
+    rule.onNodeWithText(strings.newGame).assertExists()
+    rule.onNodeWithText(strings.newGame).performClick()
+    rule.onNodeWithText(strings.newGame).assertDoesNotExist()
```

Comment:
@alexandrepiveteau Question, can we chain the action ? I read the doc, for me, it's possible to write something like this

```
rule.onNodeWithText(strings.newGame).assertExists()
     .performClick()
     .assertDoesNotExist()
```

I think it's less redundant and easier to read

Reply:
I don't know at all if it behaves exactly the same, but I agree that it's less redundant and that it's worth trying it out !